OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineActivity.java

```diff
@@ -14,7 +14,11 @@
 
 /**
  *  Top level outline for the Course
+ *
+ *  @deprecated As of release v2.13, see {@link NewCourseOutlineActivity} and
+ *  {@link NewCourseOutlineFragment} as an alternate.
```

Comment:
We can also add a `TODO` here to rename `NewCourseOutlineActivity` to `CourseOutlineActivity` once this class is deleted. Same goes for `NewCourseOutlineFragment`.